### ISSUE TITLE
example to know module type and record lim

### ISSUE THREAD

**Issue by shaurya0406** [anotherAlex154/MeccanoidForArduino#1]
can you please include an example for LIM and getmodule type

**Comment by alexfrederiksen** [comment]
Hello there. I am so sorry I took so long to respond to your issue. What kind of example would you like with them? The last two examples in the documentation go over many of the implementations. Thanks btw for looking into this project (: I didn't expect it thus I never thought to check the issues tab. /:

**Comment by shaurya0406** [comment]
Actually when I was trying LIM Mode, I was getting 0 value after each correct value
For example :
0,6,0,58,0,90,0,148,0.....
And when I was storing these values in an array I have to exclude 0 at alternate place and this took a lot of memory.
I have placed the pull up resistor as mentioned but to no avail. 
Also only one servo in a chain could operate correctly in LIM mode.

I would be very thankful if you could include an ARDUINO  example on LIM meccanoid servos

Regards

**Comment by shaurya0406** [comment]
Not an example but just fixing the issue of 0 and more than one servo in a chain

**Comment by alexfrederiksen** [comment]
Alright, so I wrote up an example and hunted for bugs. That alternating value issue is very odd and thus I'm not sure it was fixed with the code updates. Personally, my robot isn't close to operable so there wasn't the chance for testing. The example written attempts to find two servos in the first two positions in the chain and read from them in LIM mode. As it does this, it stores the values into respective buffers that are printed out at the end of a given amount of trials (default 50). I did this to see data from each servo individually. When you get the chance, try out the new test code and lemme know of your results.

Fingers crossed (;
